usb/test_app/pytest_usb_host.py

```diff
@@ -51,3 +54,56 @@ def test_usb_host(dut: Tuple[IdfDut, IdfDut]) -> None:
 
     # 3.4 Run HID tests
     host.run_all_single_board_cases(group='hid_host')
+
+@pytest.mark.esp32s2
+@pytest.mark.esp32s3
+@pytest.mark.usb_device
+def test_usb_device(dut) -> None:
```

Comment:
Here it seems that we add tusb_device logic testing in file with `pytest_usb_host.py` name. 
Is it possible to create another *.py file for device logic testing? 
Or at least rename that one to something more common, like: `pytest_usb.py`?  :thinking:

Reply:
True, will improve the tests